starknet_py/net/l1/contracts.py

```diff
@@ -36,7 +37,7 @@ def __init__(
         )
 
     def l2_to_l1_messages(
-        self, msg_hash: bytes, block_number: EthBlockIdentifier = None
+        self, msg_hash: bytes, block_number: typing.Optional[EthBlockIdentifier] = None
```

Comment:
Why not import `Optional` and use it without `typing.`?

Reply:
This module used typing as module before, I didn't want to change more lines than necessary.

Reply:
I'd change it anyways, it'd be more in line with what are we doing in pretty much every other file in the package.